cmd/verify/verify.go

```diff
@@ -40,40 +55,110 @@ func main() {
 	forever := make(chan bool)
 
 	log.Info("starting verify service")
-	var message Message
-	// var header []byte
 
 	go func() {
 		for d := range broker.GetMessages(mq, config.Broker.Queue) {
 			log.Debugf("received a message: %s", d.Body)
-			err := json.Unmarshal(bytes.Replace(d.Body, []byte(`'`), []byte(`"`), -1), &message)
-			if err != nil {
+			var m Message
+			e := json.Unmarshal(bytes.Replace(d.Body, []byte(`'`), []byte(`"`), -1), &m)
```

Comment:
Why are you replacing single quotes with double quotes? If it's a valid JSON, it should be already with double-quotes. If it has single quotes, it should be treated as a malformed JSON, AFAIK.

Reply:
That was  necessary in February but apparently not now, might have to do with the update to GO 1.14